python/paddle/fluid/contrib/slim/quantization/quantization_pass.py

```diff
@@ -1701,7 +1701,7 @@ def apply(self, graph):
                         quant_var_node = dequantized_vars_map[arg_name]
                     else:
                         quant_var_node, _ = \
-                            self._inser_quant_dequant_moving_average_abs_max_op(
```

Comment:
Don't modify this file, which is irrelevant to this PR.

Reply:
Made it as before.